pkgs/development/haskell-modules/configuration-ghcjs-8.x.nix

```diff
@@ -24,6 +24,13 @@ self: super:
     aeson = self.aeson_1_5_6_0;
   });
 
+  # Some Hackage packages reference this attribute, which exists only in the
+  # GHCJS package set. We provide a dummy version here to fix potential
```

Comment:
This is the definition of the ghcjs package set?

Reply:
Oh hmm yeah even what happened here? Taking a better look